views/account/profile/main.js

```diff
@@ -0,0 +1,56 @@
+// JQuery Datatables Initialization
+$(document).ready(function () {
+  var table = $("#casual-table").DataTable();
+  new $.fn.DataTable.Responsive(table);
+});
+$(document).ready(function () {
+  var table = $("#professional-table").DataTable();
+  new $.fn.DataTable.Responsive(table);
+});
+
+let casual_table = document.getElementById("casual-table-body");
+let professional_table = document.getElementById("professional-table-body");
+
+// These are dummy values for now; we would need to pull the casual and professional info from the db
+let casual_data_labels = ["Instagram", "Facebook"];
+let casual_data_values = ["lelekaz", "Leah Kazenmayer"];
+
+let professional_data_labels = ["Email", "LinkedIn"];
+let professional_data_values = [
+  "kazenml1@tcnj.edu",
+  "https://www.linkedin.com/in/leah-kazenmayer/",
+];
+
+function populate_table(table, labels, values) {
+  for (let i = 0; i < labels.length; i++) {
+    let tr = document.createElement("tr");
+    let label = document.createElement("td");
+    let value = document.createElement("td");
+    let action = document.createElement("td");
+    let edit_btn = document.createElement("button");
+
+    edit_btn.innerText = "Edit";
```

Comment:
For the delete button, it may be good to have a modal pop-up that asks for a confirmation, and if they click yes again, then an API request can be sent to the delete endpoint once that is set up.

For the edit button, we are going to need some kind of modal pop-up for editing both of these values (label and link) which will eventually submit to the API endpoint for updating.